tests/test_hsm.py

```diff
@@ -507,6 +590,101 @@ def testHsmShape(self):
 
         self.assertEqual(nFail, 0, "\n"+msg)
 
+    @lsst.utils.tests.methodParameters(
```

Comment:
Use `methodParametersProduct` instead of `methodParameters`. Then you wouldn't have to do the `dict` of `list` of `zip` etc.

Reply:
That's really convenient. Done!